application/src/main/java/org/togetherjava/tjbot/config/FeatureBlacklist.java

```diff
@@ -32,4 +38,24 @@ public FeatureBlacklist(Set<T> featureIdentifierBlacklist) {
     public boolean isEnabled(T featureId) {
         return !featureIdentifierBlacklist.contains(featureId);
     }
+
+    /**
+     * Filters features stream to only having enabled features and logs any disabled features
+     *
+     * @param features the feature stream to be filtered
+     * @param idExtractor function to get the class name of an object
+     * @return stream of features that are enabled
+     */
+    public <F> Stream<F> disableMatching(Stream<F> features, Function<? super F, ? extends T> idExtractor) {
```

Comment:
Could we give this method a more concise name? Features don't seem to be disabled here, just filtered and logged.